app/main.py

```diff
@@ -1,5 +1,70 @@
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int,
+            coords: list[int] | None = None
+    ) -> None:
+        self.coords = coords if coords is not None else [0, 0]
```

Comment:
```suggestion
        self.coords = coords or [0, 0]
```
it seems more simple